packages/logistration/src/LoginRedirect.jsx

```diff
@@ -1,5 +1,6 @@
 import PropTypes from 'prop-types';
 import { useParams } from 'react-router-dom';
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
This will likely go away once `@edx/frontend-platform` is added back as a peer dependency.